ccmlib/node.py

```diff
@@ -1867,17 +1866,23 @@ def _get_load_from_info_output(info):
 
 
 def _grep_log_for_errors(log):
-    matchings = []
-    it = iter(log.splitlines())
-    for line in it:
-        is_error_line = ('ERROR' in line
-                         and 'DEBUG' not in line.split('ERROR')[0])
-        if is_error_line:
-            matchings.append([line])
-            try:
-                it, peeker = itertools.tee(it)
-                while 'INFO' not in next(peeker):
-                    matchings[-1].append(next(it))
-            except StopIteration:
-                break
-    return matchings
+    def log_line_category(line):
+        match = re.search(r'(INFO|DEBUG|WARN|ERROR)', line)
+        return match.group(0) if match else 'ERROR_EXTRA'
```

Comment:
ok, so to determine if a line is an error continuation you want me to check for None rather than 'ERROR_EXTRA'? just want to make sure I got that right.